README.md

```diff
@@ -64,29 +64,17 @@ After the connection is established, you can display elements on the screen.
 
 To display elements on the aRdent glasses' screen, you need to create a `GYWDrawing` object and pass it to the `send_drawing(drawing)` method of the connected `BTDevice` object. There are three types of `GYWDrawing` objects:
 
-### 1. BlankScreen
-
-A `BlankScreen` object is a blank screen that can be used to reset the display. You can create a `BlankScreen` object like this:
-
-```python
-from pygyw.layout import drawings
-
-ws = drawings.BlankScreen(color="ff0000ff")  # ARGB
-```
-
-The color is optional and if not specified, the screen will be filled with the last color used.
-If the color was never specified, it fills the screen with white.
-
-### 2. TextDrawing
+### 1. TextDrawing
 
 A `TextDrawing` object is used to display text on the screen. You can create a `TextDrawing` object like this:
 
 ```python
-from pygyw.layout import drawings, fonts
+from pygyw.layout import drawings, fonts, color
 
-text = "Hello, world!"
-font = fonts.GYWFonts.LARGE
-td = drawings.TextDrawing(text=text, left=100, top=100, font=font, size=42, color="ff000000")
+drawing = drawings.TextDrawing(
+    text="Hello, world!", left=100, top=100,
+    font=fonts.GYWFonts.ROBOTO_MONO_BOLD, size=32,
+    color=color.Color.from_hex("000000ff"))
```

Comment:
I like to use it that way so that I don't have to manually specify every element import, and so that I avoid import conflict, but it's clearly a preference choice